tests/test_pmtypes.py

```diff
@@ -384,3 +384,24 @@ def test_operators_of_extension_local_value(self):
         self.assertNotEqual(inst1, inst2)
         self.assertFalse(inst1 == inst2)
         self.assertTrue(inst1 != inst2)
+
+
+    def test_element_with_text_list(self):
+        """Verify that a ElementWithTextList.text is a list even if text of node is None or element does not exist."""
+        node = Element('foo')
+        node.text = 'abc def ghi'
+        obj = basetypes.ElementWithTextList.from_node(node)
+        self.assertEqual(obj.text, ['abc', 'def', 'ghi'])
+
+        node.text = None
+        obj = basetypes.ElementWithTextList.from_node(node)
+        self.assertEqual(obj.text, [])
+
+        # test the case that the element that is supposed to contain the text does not exist.
+        obj = basetypes.ElementWithTextList()
+        with unittest.mock.patch.object(xml_structure.NodeTextListProperty,
+                                        '_get_element_by_child_name',
+                                        new=unittest.mock.MagicMock(side_effect=xml_structure.ElementNotFoundError)):
+
+            obj.update_from_node(node)
+            self.assertEqual(obj.text, [])
```

Comment:
I suggest that you assert that the mock has been called
```python
... as mocked:
    ...
    mocked.assert_called_once()
```

Reply:
done